code/modules/mob/living/silicon/silicon.dm

```diff
@@ -34,7 +34,7 @@
 	var/hackedcheck[1]
 	var/devillawcheck[5]
 
-	var/sensors_on = 0
+	var/sensors_on = TRUE //are our HUDs on?

```

Comment:
do the autodoc

Reply:
?

All of the other PRs I've seen just leave comments like this one; what should I be doing differently here?

Reply:
It's for the [autodoc](http://codedocs.tgstation13.org/mob/living/silicon.html) page. All variables and procs are supposed to be documented using a specific format when they are declared. For example on a somewhat populated page, see the [mecha documentation](http://codedocs.tgstation13.org/obj/mecha.html).

Autodoc with three slashes before the var.

```
///Variable that holds the status of the silicon hud overlay. TRUE for enabled, FALSE for disabled.
var/sensors_on = TRUE